sources/api/netdog/src/main.rs

```diff
@@ -60,12 +71,6 @@ struct LeaseInfo {
     dns_search: Option<Vec<String>>,
 }
 
-#[derive(Debug, PartialEq, Deserialize)]
-#[serde(rename_all = "kebab-case")]
-enum InterfaceName {
```

Comment:
This was the only thing I was a little leery of.  However, considering that wicked only calls `install` for interfaces that we are explicitly configuring it seems like this guard is unnecessary.